app/templates/introduction.html

```diff
@@ -16,7 +16,13 @@
   {% block business_details %}
 
     <div class="u-mt-s">
-      <h2 class="saturn">Your response is legally required</h2>
+      <h2 class="saturn">
+        {%- if legal_basis == 'StatisticsOfTradeAct' -%}
+          Your response is legally required
+        {%- else -%}
+          Your response to this survey is of great value to government, whether or not you carry out innovation activities
```

Comment:
This line is specific to the UKIS survey, rather than being the default for non StatisticsOfTradeAct surveys, so should probably be in the UKIS introduction.html?